pkgs/applications/editors/vim/plugins/overrides.nix

```diff
@@ -920,6 +920,10 @@ self: super: {
     dependencies = with self; [ telescope-nvim plenary-nvim ];
   };
 
+  ollama-nvim = super.ollama-nvim.overrideAttrs {
+    dependencies = with super; [ plenary-nvim ];
```

Comment:
good reflex to hink about the override !  as there is only one dependency super.plenary-nvim would have been shorter. In fact I also  think it should use the final plenary-nvim in case it's customized, so that would be self.plenary-nvim.

Reply:
Thank you for pointing this out !
Fixed :)